docs/modules/user-guide/partials/proc_adding-a-registry-schema.adoc

```diff
@@ -0,0 +1,423 @@
+[id="adding-a-registry-schema_{context}"]
+= Adding a registry schema
+
+[role="_abstract"]
+To add samples and stacks from other repositories, put the file, `extraDevfileEntries.yaml`, under the root of a devfile registry. This file contains the location information of the extra samples and stacks.
+
+.extraDevfileEntries.yaml schema
+* samples: schema[] - the sample list
```

Comment:
I would format each of these elements in monospace (both the name of each element as well as its type). The description does not need to be formatted. For example:

```adoc
* `samples`: `schema[]` - the sample list
```

To be honest, though, I think the formatting of this list could be better represented with a table. For the headings, we include "key" or "name", "type", and "description". I believe this would be significantly easier to read and parse.

```adoc
[cols="3*"]
|===
|Name |Type |Description

|`samples`
|`schema[]`
| The sample list.

|===
```